templates/main.html

```diff
@@ -1,22 +1,33 @@
 <!DOCTYPE html>
-<html lang="en">
+<html>
   <head>
     <meta charset="utf-8">
     <title>GCI Leaders</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
```

Comment:
This looks like it is doing something unrelated to the issue you are solving..?
please correct me if I am wrong about that.

Reply:
Looks like it was related to https://github.com/coala/gci-leaders/pull/78/files#diff-c8935f416a66e3b982a42dccd8206926L6